test/sql/functap.sql

```diff
@@ -64,6 +64,14 @@ SELECT * FROM check_test(
     ''
 );
 
+SELECT * FROM check_test(
+    has_function( 'pg_catalog', 'now', ARRAY[''] ),
```

Comment:
This not an empty array. It's a text array with a single item in it. To make an empty array, use `'{}'::text[]`.